google/cloud/internal/backoff_policy.cc

```diff
@@ -25,9 +25,7 @@ std::unique_ptr<BackoffPolicy> ExponentialBackoffPolicy::clone() const {
 }
 
 std::chrono::milliseconds ExponentialBackoffPolicy::OnCompletion() {
-  using std::chrono::duration_cast;
-  using std::chrono::microseconds;
-  using std::chrono::milliseconds;
+  using DoubleMicroseconds = std::chrono::duration<double, std::micro>;
```

Comment:
Opt: If you make it a class-private alias instead, you can use it for the declarations of the data members too.

Reply:
Done